CHANGELOG.md

```diff
@@ -6,14 +6,95 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [holochain\_sqlite-0.2.0](crates/holochain_sqlite/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p-0.2.0](crates/kitsune_p2p/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_proxy-0.2.0](crates/kitsune_p2p_proxy/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_transport\_quic-0.2.0](crates/kitsune_p2p_transport_quic/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_mdns-0.2.0](crates/kitsune_p2p_mdns/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_fetch-0.2.0](crates/kitsune_p2p_fetch/CHANGELOG.md#0.2.0)
+
+## [holochain\_keystore-0.2.0](crates/holochain_keystore/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_types-0.2.0](crates/kitsune_p2p_types/CHANGELOG.md#0.2.0)
+
+## [mr\_bundle-0.2.0](crates/mr_bundle/CHANGELOG.md#0.2.0)
+
+## [holochain\_util-0.2.0](crates/holochain_util/CHANGELOG.md#0.2.0)
+
+## [hdk-0.2.0](crates/hdk/CHANGELOG.md#0.2.0)
+
+## [holochain\_zome\_types-0.2.0](crates/holochain_zome_types/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_dht-0.2.0](crates/kitsune_p2p_dht/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_block-0.2.0](crates/kitsune_p2p_block/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_bin\_data-0.2.0](crates/kitsune_p2p_bin_data/CHANGELOG.md#0.2.0)
+
+## [hdi-0.3.0](crates/hdi/CHANGELOG.md#0.3.0)
+
+## [hdk\_derive-0.2.0](crates/hdk_derive/CHANGELOG.md#0.2.0)
+
+## [holochain\_integrity\_types-0.2.0](crates/holochain_integrity_types/CHANGELOG.md#0.2.0)
+
+## [kitsune\_p2p\_timestamp-0.2.0](crates/kitsune_p2p_timestamp/CHANGELOG.md#0.2.0)
+
+## [holo\_hash-0.2.0](crates/holo_hash/CHANGELOG.md#0.2.0)
+
+- Adds more ways to convert between different hash types [\#2283](https://github.com/holochain/holochain/pull/2283)
+  - Adds `.into_agent_pub_key() -> Option<AgentPubKey>` for `AnyDhtHash` and `AnyLinkableHash`
+  - Adds `TryFrom` impls for all fallible conversions. For instance, if you have a link target (of type AnyLinkableHash), you can now do `let entry_hash: EntryHash = link.target.try_into().unwrap()` if you expect the link target to be an entry hash. (Though we don’t recommend using `.unwrap()` in real code\!)
+
+## [kitsune\_p2p\_dht\_arc-0.2.0](crates/kitsune_p2p_dht_arc/CHANGELOG.md#0.2.0)
+
+## [holochain\_trace-0.2.0](crates/holochain_trace/CHANGELOG.md#0.2.0)
+
+## [fixt-0.2.0](crates/fixt/CHANGELOG.md#0.2.0)
+
 # 20230426.003734
 
 This release features  more secure version of Holochain that incorporates advanced network security protocols using WebRTC End-to-End Encryption and addresses known Holochain security vulnerabilities.
 
 It also introduces new functionality and improves some of the existing:
-* app-level blocking, for when an application wants to allow agents to block each other for reasons other than validation failures
-* user experience improvements that provide better network information
-* and improved system validation to make the network more secure and reliable.
+
+- app-level blocking, for when an application wants to allow agents to block each other for reasons other than validation failures
```

Comment:
ops i added this under the previous release heading instead of unreleased, but i just pushed a fix to develop